pkg/server/server.go

```diff
@@ -216,6 +216,12 @@ func newCNIServer(rundir string, kubeClient *k8s.ClientInfo, exec invoke.Exec, s
 			}
 		}))).Methods("POST")
 
+	router.HandleFunc(api.MultusHealthAPIEndpoint, promhttp.InstrumentHandlerCounter(s.metrics.requestCounter.MustCurryWith(prometheus.Labels{"handler": api.MultusHealthAPIEndpoint}),
```

Comment:
@maiqueb I'm not clear your comment. Could you please clarify it?

Reply:
Do we need metrics for this endpoint ? @s1061123

Reply:
hmm... it is good question. I'd like to have your opinion, first. What do you think about it?

Reply:
I don't think it makes much sense. But on the other hand, it makes the API consistent.

Reply:
I think about it for a while and I suppose we keep this metrics. One reason is not so big overhead and another reason it might be used (not completely useless).